app/controllers/events_controller.rb

```diff
@@ -170,6 +170,8 @@ def render_ical(entries)
 
   def for_typeahead(entries)
     entries.map do |entry|
+      entry.reload
+      entry = entry.becomes(entry.type.constantize)
```

Comment:
Das habe ich noch nicht ganz verstanden und wäre froh um Insights von einem Lead-Dev:

`entry` enthält nur ganz wenige Attribute, vermutlich aus Performance-Gründen. Wo wird das genau festgelegt, welche Attribute geladen werden? Vermutlich würde es reichen, auch das `type` Attribut zu laden, damit die STI-Mechanismen greifen. In diesem Fall wäre das wichtig, weil sonst nicht die richtigen RoleTypes geladen werden.